VideoWeb/VideoWeb/ClientApp/src/index.html

```diff
@@ -29,10 +29,11 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="assets/images/favicon.ico" />
+    <link href="assets/fontawesome/css/all.css" rel="stylesheet">
+    <!-- <script defer src="assets/fontawesome/all.js"></script> -->
```

Comment:
This and all.js could be removed before merging.

Reply:
The js is another option instead of storing css and webfonts. It works with the same classes but instead of importing fonts it draws them as svgs.